js/api.js

```diff
@@ -0,0 +1,20 @@
+const URL = 'https://28.javascript.pages.academy/kekstagram';
+
+const load = async () =>
+  fetch(`${URL}/data`)
+    .then((response) => response.json())
+    .catch((reject)=> reject());
+
+const setSubmit = async (data) => {
```

Comment:
Убираем все и сразу возвращаем Promise, обрабатываем ошибки.
 Для примера я тут вызываю showAlert но надо показывать формы по тз, т.е. завести отдельный модуль для них и показывать попап успешной отправки или ошибки
```suggestion
const submit = async (data) =>
  fetch(
    URL + 3,
    {
      method: 'POST',
      body: data,
    })
    .then((response) => {
      if (response.ok) {
        showAlert('Данные успешно отправились!'); // Это заменить на показ формы успешной отправки
        return Promise.resolve();
      }
      throw new Error('Данные не отправились');
    })
    .catch((reason) => {
      showAlert(reason.message); // Это заменить на показ формы неуспешной отправки
      return Promise.reject(reason.message);
    });

```